util/walk/walk.go

```diff
@@ -447,6 +456,13 @@ func createPathScanner(path string) *bufio.Scanner {
 
 			if i+width < len(data) {
 				next, _ := utf8.DecodeRune(data[i+width:])
+				// case: escape chars
+				_, escapeNext := escape[next]
+				if r == '\\' && escapeNext {
+					// Skips the next character
+					i += width
+					continue
+				}
 				if isValidSyntax(r, next) {
```

Comment:
You need to check in this function that a backslash (`\`) is followed by either another backslash or the characters that can be escaped. In other words, you should also escape the backslash character. `path\to\element` should be an invalid path, while `path\\to\\element` should be valid.

This will create a behavior that is more intuitive and closer to what developers are used to when escaping characters in general.

This function is badly named, it should be `isSyntaxInvalid` or something like that.